README.md

```diff
@@ -304,6 +305,65 @@ const transactionHash = getCredentialDeploymentTransactionHash(credentialDeploym
 const transactionStatus = await client.getTransactionStatus(transactionHash);
 ```
 
+## Construct identityInput for creating credentials
+
+When creating a new identity, the user will choose an identity provider, create an id-use-data object, which contains the private data to use the identity, and obtain an identity object from the identity provider.
+
+To create accounts/credentials on that identity, this SDK expects an "IdentityInput" object, which contains the identity object, the id-use-data, and the identity provider's information.
+
+Below is an example of how to construct the identityInput, with a plantext id-use-data.json from the [user-cli guide](https://github.com/Concordium/concordium-base/blob/main/rust-bins/docs/user-cli.md#generate-a-request-for-the-identity-object), and an id-object file.
```

Comment:
Perhaps add a subsection heading here to delineate this example from the one below.